Tests/Element/CoordinatesUtilityTest.php

```diff
@@ -13,14 +12,22 @@
 
 class CoordinatesUtilityTest extends WebTestCase
 {
+    const SRS_NAME = "EPSG:4647";
+    const SRS_TITLE = "ETRS89 / UTM zone N32";
+    const SRS_DEFINITION = "+proj=tmerc +lat_0=0 +lon_0=9 +k=0.9996 +x_0=32500000 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs";
+
     /**
      * @var CoordinatesUtility
      */
     private $coordinatesUtility;
 
     public function setUp()
     {
-        $container = new Container();
+        $client = static::createClient();
```

Comment:
it seems, that this variable is not used

Reply:
yeap, removed